SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/CrossServerRPCService.cpp

```diff
@@ -212,13 +232,19 @@ void CrossServerRPCService::PopulateDataStore(const Worker_EntityId EntityId)
 	Schema_ComponentData* SenderACKData =
 		Entity.Components.FindByPredicate(ComponentIdEquality{ SpatialConstants::CROSSSERVER_SENDER_ACK_ENDPOINT_COMPONENT_ID })
 			->GetUnderlying();
-	Schema_ComponentData* ReceiverData =
-		Entity.Components.FindByPredicate(ComponentIdEquality{ SpatialConstants::CROSSSERVER_RECEIVER_ENDPOINT_COMPONENT_ID })
-			->GetUnderlying();
+
+	const ComponentData* ReceiverData =
+		Entity.Components.FindByPredicate(ComponentIdEquality{ SpatialConstants::CROSSSERVER_RECEIVER_ENDPOINT_COMPONENT_ID });
 
 	CrossServerEndpoints& NewEntry = CrossServerDataStore.FindOrAdd(EntityId);
 	NewEntry.ACKedRPCs.Emplace(CrossServerEndpointACK(SenderACKData));
-	NewEntry.ReceivedRPCs.Emplace(CrossServerEndpoint(ReceiverData));
+
+	// The worker entity won't have a receiver
```

Comment:
Ah, I made a comment to this effect earlier in this file - can we remove this edge case?